src/style.css

```diff
@@ -0,0 +1,101 @@
+/* #region fonts */
+@font-face {
+  font-family: avenir;
+  src: url(fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: heavy;
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: book;
+  font-weight: normal;
+}
+/* #endregion */
+*:focus {
+  outline: none;
+}
+
+body {
+  font-family: avenir, sans-serif;
+}
+
+.form {
+  position: relative;
```

Comment:
Input will broke if u add some element to top of the form, let's add relative to the input wrapper, `label` for example will be good

Reply:
Added